posthog/models/remote_config.py

```diff
@@ -177,8 +179,12 @@ def build_config(self):
                 config["quotaLimited"] = ["recordings"]
                 config["sessionRecording"] = False
 
-        config["surveys"] = True if team.surveys_opt_in else False
         config["heatmaps"] = True if team.heatmaps_opt_in else False
+        surveys_response = get_surveys_response(team)
+        config["surveys"] = surveys_response["surveys"]
```

Comment:
The JS SDK expects a boolean. Will that work with the new backend and the old JS SDK version? (not everyone uses the loader script).

Reply:
https://github.com/PostHog/posthog-js/blob/99e92b5f01858f4fc4584e8c6b45269ea432d8e1/src/posthog-surveys.ts#L73
at least the key is being read here!